backend/src/v2/test/Makefile

```diff
@@ -9,19 +9,9 @@ include $(ENV_PATH)
 SHELL = /bin/bash
 
 .PHONY: integration-test
-integration-test: upload
+integration-test:
 	export KF_PIPELINES_ENDPOINT=$(HOST) \
-		&& python -u sample_test.py \
-		--samples_config samples/test/config-integration.yaml \
-		--timeout_mins 60 \
-		--context $(GCS_ROOT)/src/context.tar.gz \
-		--gcs_root $(GCS_ROOT)/data \
-		--gcr_root $(GCR_ROOT) \
-		--kfp_package_path "$(KFP_PACKAGE_PATH)"
-
-.PHONY: upload
-upload: context
-	python -u scripts/upload_gcs_blob.py tmp/context.tar.gz $(GCS_ROOT)/src/context.tar.gz
+		&& python -u ../../../../samples/v2/sample_test.py
 
 .PHONY: context
 context:
```

Comment:
This file adds nothing to the logic and can be completely removed. You can invoke `sample_test.py` from `backend/src/v2/test/integration-test.sh`.

Reply:
Well, I decided not to change the scope of things. Those files probably are going to change again once we starting moving to our CI. For now, I kept as closer to the original. Any problem?

Reply:
I would remove it anyway. But feel free to decide it.